custom_components/powercalc/data/lidl/HG06467/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.9.6:docker"
+    },
+    "name": "Melinera HG06467",
```

Name will be used in second column to generate supported models documentation. Best to use the marketed name, and not duplicate the model id there.

```suggestion
    "name": "Melinera Christmas Lights",
```